flake.nix

```diff
@@ -103,7 +103,7 @@
               # tools that work only with default compiler
               fourmolu = "0.10.1.0";
               hlint = "3.5";
-              haskell-language-server = "2.0.0.0";
+              haskell-language-server = { src = nixpkgs.haskell-nix.sources."hls-2.0"; };
```

Comment:
Just curious what the reason for this is?  Is it working better like this?

Reply:
Honestly, I have no idea. This is what @hamishmack did in this PR: https://github.com/input-output-hk/cardano-ledger/pull/3479

So, I just copied it over.

If I had to guess this will use the latest 2.0 version from haskell-nix-sources, instead of us bumping it manually for every non-breaking release. For example `2.0.0.1` is needed for ghc-9.2.8 and 9.6.2.